docs/pages/agents/build-an-agent.mdx

```diff
@@ -0,0 +1,80 @@
+# Build an agent that uses XMTP
+
+This guide covers how to build agents that communicate with humans and other agents in chats that provide all of [XMTP's messaging security properties](/protocol/security).
+
+## Resources to get started
+
+### Quickstart video guide
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/djRLnWUvwIA?si=JX25iQt57wgXnqVX" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+### Agent examples
+
+Explore and run example agents built with the XMTP Node SDK in the [xmtp-agent-examples](https://github.com/ephemeraHQ/xmtp-agent-examples) repo.
```

Comment:
should we link to the Node SDK here? could link to the repo, or the NPM registry.